src/main/java/com/ctc/wstx/sax/WstxSAXParserFactory.java

```diff
@@ -50,6 +50,8 @@ public class WstxSAXParserFactory
     public WstxSAXParserFactory()
     {
         this(new WstxInputFactory());
+
+        mStaxFactory.getConfig().doReplaceEntityRefs(false);
```

Comment:
Could you add a comment with date, ref to the issue, denoting when change was made.

Reply:
Sure, done